From bf2f298cd29e2ae225a1ddacdabc19a20ba28b89 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Fri, 31 Jan 2020 21:07:10 +0530
Subject: [PATCH 1087/1851] net: macb: Initialize bool to false instead of zero

commit 34756da361f874a71e873720ae6b25931f1fa704 from
https://github.com/Xilinx/linux-xlnx.git

Warning:
drivers/net/ethernet/cadence/macb_main.c:1761:14-20: WARNING: Assignment
of bool to 0/1

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/net/ethernet/cadence/macb_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/cadence/macb_main.c b/drivers/net/ethernet/cadence/macb_main.c
index e433e1ea6a09..27aed5e12b2a 100644
--- a/drivers/net/ethernet/cadence/macb_main.c
+++ b/drivers/net/ethernet/cadence/macb_main.c
@@ -1827,7 +1827,7 @@ static netdev_tx_t macb_start_xmit(struct sk_buff *skb, struct net_device *dev)
 	unsigned long flags;
 	unsigned int desc_cnt, nr_frags, frag_size, f;
 	unsigned int hdrlen;
-	bool is_lso, is_udp = 0;
+	bool is_lso, is_udp = false;
 	netdev_tx_t ret = NETDEV_TX_OK;
 
 	if (macb_clear_csum(skb)) {
-- 
2.31.1

